test/jasmine/tests/shapes_test.js

```diff
@@ -75,33 +159,32 @@ describe('Test shapes:', function() {
 
     describe('Plotly.relayout', function() {
         it('should be able to add a shape', function(done) {
-            var pathCount = countShapePaths();
+            var pathCount = countShapePathsInUpperLayer();
             var index = countShapes(gd);
             var shape = getRandomShape();
 
-            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(function() {
-                expect(countShapeLayers()).toEqual(1);
-                expect(countShapePaths()).toEqual(pathCount + 1);
+            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(function()
+            {
```

Comment:
Ah good point. Don't worry about going over 80 characters. I don't think we'll ever enforce that rule in our jasmine test suites.